entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
This file is no longer necessary, but I included it as an example if we wanted a simplified build process (without the yaml file)